main.nf

```diff
@@ -253,7 +252,6 @@ if (params.sentieon) log.warn "[nf-core/sarek] Sentieon will be used, only works
 ================================================================================
 */
 
-
 include { BWAMEM2_MEM }           from './modules/local/bwamem2_mem.nf'
```

Comment:
```suggestion
include { BWA_MEM2_MEM }           from './modules/local/bwa_mem2_mem.nf'
```

Reply:
Would have been nice if they were able to add this directly to BWA somehow! Means we will need a `bwa` and `bwa_mem2` software folder now...